meetings/2021-03-10.rst

```diff
@@ -0,0 +1,98 @@
+======================
+Meeting March 10, 2021
+======================
+
+- With no invitee or particular dedicated topic, the discussion was broadly centered around the present state of the effort, our aims, and how to continue the effort.
+
+- Use of "structure" vs. "crystal" (a discussion started last time, but cut short.)
+
+  * This topic is related to if the 'single structure materials model' in our prototype ontology should be 'single crystal materials model.'
+
+  * The underlying concept described by a crystal is something with an underlying periodicity.
+    It may be non-periodic in the absolute sense due to defects, disorder, etc., but these are modifications onto a system where Bloch's theorem applies.
```

Comment:
I only realized this as I was writing up these notes; the prime issue with moving over from "single structure" to "singe crystal" is that at least in OPTIMADE structures (vs. crystals) can have non-periodic dimensions. This means that even a molecule can be a structure, and I think it would not be correct to call that system a crystal.

So, either we need to allow, e.g., molecules to be included under "single crystal materials model", OR, we need to separate out systems with fewer periodic dimensions out of the "single crystal materials model". We probably should pick this discussion up again next meeting - and/or - it may be something for the intended "single crystal materials model" working group to look at.